src/deleteTask.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/no-cycle
+import { createList } from './addtask.js';
+
+export default function deleteTask(items) {
```

Comment:
- [] Please try to use the newer es6 syntax in situations where it is possible. It is recommended to use the newer arrow function syntax.
something like this would be great
```suggestion
export default (items) => {
```